int32 uav_id
int32 dst_id